### DISCUSSION TITLE
Sharing ArchR object

### DISCUSSION THREAD

**Discussion by jindalk** [GreenleafLab/ArchR#1702] [Questions / Documentation] [answered]
Hi,
Is there a preferred strategy for sharing entire ArchR objects? Only way I could think of is downloading an ArchR project folder to a new location and then running `subsetArchRProject` with all cells to reset internal paths?

Thanks

**Comment by rcorces** [comment, marked as answer]
You should save your desired ArchRProject using `saveArchRProject()` and then share the entire resulting directory (indicated by the `outputDirectory` param.
https://www.archrproject.com/reference/saveArchRProject.html

You can tar.gz the folder to save some space and create a single file to share.

**Reply by rcorces** [reply]
Oh, and the person receiving the project would un-tar.gz the folder and then load the project using `loadArchRProject()`
